feature-detects/css/remunit.js

```diff
@@ -25,7 +25,7 @@ define(['Modernizr', 'createElement'], function( Modernizr, createElement ) {
     try {
       div.style.fontSize = '3rem';
     }
-    catch( er ) {}
+    catch ( er ) {}
```

Comment:
Interesting.. I don't know why but I had nulled the `disallowSpacesInsideParentheses` rule in the PR haha.